examples/cpp/reflection/reflection_client.cc

```diff
@@ -0,0 +1,216 @@
+/*
```

Comment:
This file is put here to demonstrate the use of ProtoReflectionDescriptorDatabase. It will be removed before checking in server reflection. It has been reformatted and added to grpc_tool as `grpc_tool ls -l <address>` command.